src/components/Addtask/index.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import { addTask } from "/src/store/actions";
+import { sanitizeText } from "/src/utils/helpers/sanitizeText"
+import "./index.scss";
+
+const AddTask = ({ isFormOpen, setIsFormOpen }) => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState("");
+  const [error, setError] = useState(null);
+
+  const handleSubmit = (e) => {
```

Comment:
```suggestion
  const handleSubmit = (event) => {
```

Reply:
is this changes missed by any chance? Why is it still here?